proto/caraml/upi/v1/upi.proto

```diff
@@ -132,3 +108,14 @@ message ModelMetadata {
   // Model version that produce prediction
   string version = 2;
 } 
+
+// Transformer input contains additional information that can be used to enrich prediction_table using standard transformer.
+// All tables and variables within transformer input will be imported to the standard transformer runtime automatically.
+message TransformerInput {
+   // List of tables
+   // All tables must have unique name.
+   // Each table doesn't need to have same number of row.
+   repeated Table tables = 1;  
+   // List of variables
+   repeated NamedValue variables = 2;
```

Comment:
do you see concrete use cases for `variables`? Might it be simpler to make everything a table?

Reply:
I think there are valid cases to declare scalar variable based on current usage of standard transformer. 
> Might it be simpler to make everything a table?
It is simpler but by not much.